src/plugins/stacked-barchart/.gitignore

```diff
@@ -0,0 +1,30 @@
+# Logs
```

Comment:
Not sure this file is required.

Reply:
You are right) removed this `.gitignore`, but added some lines to the .gitignore at project root to ignore `node_modules`, `dist`, and `coverage` directories.